README.md

```diff
@@ -134,6 +135,45 @@ Separate each step of the task to an individual task. For example a "generate ic
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Group related tasks by prefix 
+
+Bundle your tasks with a prefix so you can execute them all at once.
+
+### Why?
+
+* Bundling helps keeping your tasks organized.
+* Tasks grouped by prefix can be easily executed with one command.
+* Your script API remains unchanged when tasks are added, removed or renamed.
+
+### How?
+
+`package.json`:
+``` js
+/* recommended */
+scripts: {
+	"build:cleanDistDir": "rimraf dist/",
+	"build:optimizeSVG": "svgo --folder svg/",
+	"build:generatePNGFallback": "svg2png svg/*.svg"
+}
+
+/* avoid */
+scripts: {
+	"cleanDistDir": "rimraf dist/",
+	"optimizeSvg": "svgo --folder svg/ --output dist/svg",
+	"generatePNGFallback": "svg2png svg/*.svg --output dist/png"
+}
+```
+
+To run all scripts prefixed with `build:` use [npm-run-all](https://www.npmjs.com/package/npm-run-all). If you use taskrunners, see their documentation for implementation details.
```

Comment:
I don't quite understand the second sentence.
You can also add `A CLI tool to run multiple npm-scripts in parallel or sequential / serial.` as definition of npm-run-all